elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -277,6 +280,22 @@ export abstract class BaseTransaction {
 			);
 		}
 
+		// Skip for type 4 due to dynamic nature of fee for multi-signature. Validation happens on multisignature class.
+		if (
+			this.type !== MULTISIGNATURE_TYPE &&
```

Comment:
Instead of adding an exception here, what about create a method called `validateFee` inside `BaseTransaction` and overwrite it inside `MultisignatureTransaction`?

This way we provide a default fee validation (`this.fee.eq((this.constructor as typeof BaseTransaction).FEE)`) but also allow custom transactions to define its own fee validation logic if necessary.

Reply:
I thought of that but I think fee is the only exception and if we split we'll need to apply any base transaction validation change to multi-signature. I know the exception is not nice but I think is less maintenance than this suggestion. What you think about this?

Reply:
That is a good point but having a `validateFee` method has more pros than cons imo.

Reply:
Changed please check again @lsilvs